2-user.sh

```diff
@@ -17,30 +17,26 @@ Installing AUR Softwares
 # You can solve users running this script as root with this and then doing the same for the next for statement. However I will leave this up to you.
 source $HOME/ArchTitus/setup.conf
 
-cd ~
-case $AURHELPER in
-  "1")
-    git clone "https://aur.archlinux.org/yay.git"
-    cd ~/yay
-    makepkg -si --noconfirm
-    cd ~ ;;
-  "2")
-    git clone "https://aur.archlinux.org/paru.git"
-    cd ~/paru
-    makepkg -si --noconfirm
-    cd ~ ;;
-  "3")
-    echo ""
-    echo "no aur helper selected"
-    echo ""
-esac
+addAUR () {
+  cd ~
+  git clone "https://aur.archlinux.org/$1.git"
+  cd ~/$1
+  makepkg -si --noconfirm
+  cd ~
+}
+
+if [$AURHELPER -ne ""] then
+  addAUR $AURHELPER
+fi
 
 touch "~/.cache/zshhistory"
 git clone "https://github.com/ChrisTitusTech/zsh"
 git clone --depth=1 https://github.com/romkatv/powerlevel10k.git ~/powerlevel10k
 ln -s "~/zsh/.zshrc" ~/.zshrc
 
-yay -S --noconfirm --needed - < ~/ArchTitus/pkg-files/aur-pkgs.txt
+for PACKADGE in $(cat ~/ArchTitus/pkg-files/aur-pkgs.txt) do
+  addAUR $PACKADGE
+esac
```

Comment:
here you're no using the desired helper, but rather installing the package by hand no?
+ spelling issue : `PACKAGE` instead of `PACKADGE` 😉

Reply:
What's wrong with that it's just a more reliable solution that doesn't require me to mess around with add separate install functions for every helper or installing another helper and then uninstalling it

Reply:
Just fixed the spelling issue

Reply:
Nothing wrong obviously, at least not to my eyes. Was just wondering : why bother when you have a helper ready to go.

Reply:
Well mainly because I don't like having to implement multiple if statements where if you select paru or yay use it and if you select none use the custom code or temporarily install another helper where either option is less clean

Reply:
Seemed like the entire point of the change, though, is to implement an aur helper based on user preference.